autopush-common/src/util/mod.rs

```diff
@@ -11,6 +11,8 @@ pub mod user_agent;
 
 pub use self::timing::{ms_since_epoch, sec_since_epoch, us_since_epoch};
 
+pub const ONE_DAY_IN_SECONDS: u64 = 60 * 60 * 24;
```

Comment:
I'm generally in favor of explicit for stuff like this rather than mystery numbers. Plus, the compiler does the work of converting this to a single number anyway. This one may be well known enough to most folk that it doesn't matter. That said, we're not very consistent about it. Perhaps that's a good general clean-up issue.